modules/kubernetes/fluxcd-v2-azdo/main.tf

```diff
@@ -99,11 +97,6 @@ data "azuredevops_git_repository" "cluster" {
   name       = var.cluster_repo
 }
 
-resource "random_password" "cluster" {
-  length  = 32
```

Comment:
random_password is set by the application now a days if i remember correctly

Reply:
Yes, all password generation has been removed from the Terraform now.